### PR TITLE
make work \w npm v3

### PR DESCRIPTION
**Pull request by stefanpenner**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -68,6 +68,7 @@
   "dependencies": {
     "babel": "^5.8.38",
     "babel-plugin-htmlbars-inline-precompile": "0.0.5",
+    "ember-cli-legacy-blueprints": "^0.1.0",
```
Comment (Gaurav0) [inline package.json]: Does this need to be in dependencies?

Comment (stefanpenner) [inline package.json]: typically we prefer this yes, as ember-cli being dependent on ^ may change over time. But this app appears to require it

Comment (stefanpenner) [inline package.json]: that way the dependencies are truthful and explicit .

Comment (Gaurav0) [inline package.json]: Should only be required during build. Moved to devDependencies.

Comment (stefanpenner) [inline package.json]: 👍

**Pull request merged**